Whisky/Utils/Wine.swift

```diff
@@ -252,6 +252,11 @@ class Wine {
         return try runWineserver(["-k"], bottle: bottle)
     }
 
+    static func addShortcut(name: String, url: URL, bottle: Bottle) async throws {
```

Comment:
This function does not make any use of Wien and shouldn't be implemented in this class

Reply:
true, should have noted that that looks out of place

Reply:
All those classes were static. That's why I figured it fit lol. I'll push a change tonight or tomorrow.